src/main/groovy/jenkins/automation/utils/CheckmarxSecurityJobBuilder.groovy

```diff
@@ -0,0 +1,63 @@
+package jenkins.automation.utils
+
+import javaposse.jobdsl.dsl.DslFactory
+import javaposse.jobdsl.dsl.Job
+
+
+class CheckmarxSecurityJobBuilder {
+
+    String name
+    String description
+    String baseUrl
+    String scanRepo
+    String checkmarxComment
+    Boolean vulnerabilityThresholdEnabled
+    String highThreshold
+    String mediumThreshold
+    String lowThreshold
+
+    Job build(DslFactory factory){
+        factory.job(name){
+            it.description this.description
+            
+            scm {
+                git {
+                    remote {
+                        name('Repo to Scan')
+                        url(scanRepo)
+                    }
+                    
+                    branch('*/master')
+                    wipeOutWorkspace(true)
```

Comment:
So much so, that I probably wouldn't add it to the builder as a default, and leave it up for people to add that behavior for their jobs if it actually became necessary, which I highly doubt it would